deploy/kubernetes/docker/rss-env.sh

```diff
@@ -40,4 +41,23 @@ JPS="${JAVA_HOME}/bin/jps"
 # HADOOP_CONF_DIR, Hadoop configuration directory (Default: ${HADOOP_HOME}/etc/hadoop)
 # RSS_PID_DIR, Where the pid file is stored (Default: ${RSS_HOME})
 # RSS_LOG_DIR, Where log files are stored (Default: ${RSS_HOME}/logs)
+
+# Variables for the dashboard
+# DASHBOARD_JAVA_OPTS, JVM options for the dashboard, will override the default JVM options
```

Comment:
> It might be too large for coordinator, but it should be normal for ShuffleServers to have max heap size of 80G.

We have to use the new VAR with PREFIX.

```
XMX_SIZE="8g" # Coordinator/Dashboard JVM XMX size
SHUFFLE_SERVER_XMX_SIZE="00g" # ShuffleServer JVM XMX size
```